modules/couchdb/collect.go

```diff
@@ -49,6 +50,15 @@ func (CouchDB) collectSystemStats(collected map[string]int64, ms *cdbMetrics) {
 	}
 }
 
+func (CouchDB) collectActiveTasks(collected map[string]int64, ms *cdbMetrics) {
+	if !ms.hasActiveTasks() {
+		return
+	}
+	for taskIndex := range ms.ActiveTasks {
+		collected["active_tasks_"+ms.ActiveTasks[taskIndex].Type]++
```

Comment:
prefer 

```go
for i, v := range slice {
	...
}
```

over 

```go
for i := range slice {
	slice[i]
	...
}
```

and prefer `i` for index, it is de-facto standard and it is clear from the context (first item is index, we all know that)

Reply:
Fixed